server/recaptcha.go

```diff
@@ -24,7 +24,7 @@ func verifyReCaptcha(token, reCaptchaSecret, action, remoteIP string) error {
 	}
 	resp, err := http.PostForm(RECAPTCHA_URL, params)
 	if err != nil {
-		log.Fatal().Err(err).Msg("error occurred during making recaptcha request")
+		logger.Fatal("error occurred during making recaptcha request", zap.Error(err))
```

Comment:
I was thinking that you removed it because it shouldn't be _fatal_, which I think I agree with, but it still seems loggable, maybe as `Warn`? Does that make sense? `Fatal` seems extreme - since the error isn't necessarily our fault or in our control.

Reply:
Yep (actually makes sense to remove recaptcha since it's not used)